cli/pkg/up/build.go

```diff
@@ -62,10 +61,7 @@ func LoadRiofile(path string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-		var r map[string]interface{}
-		if err := yaml.Unmarshal(content, &r); err == nil {
-			return content, nil
-		}
+		return content, nil
```

Comment:
We also need this workflow to work:
If `rio up -f Dockerfile`, use the defaultRiofileContent. This is just for convenience if user pass a dockerfile.(darren want this feature).

Reply:
So we might have to apply answer first and check if it is a yaml manifest, otherwise we treat it as Dockerfile and will fall back to look for Riofile.

Reply:
Check `LoadRiofile` now and see if you think that will work for all cases

Reply:
If you think that works we should prob add a unit test